applications/aero_acoustic/co_rotating_vortex_pair/application.h

```diff
@@ -536,13 +536,26 @@ class Application : public ApplicationBase<dim, Number>
         refine_triangulation_around_center(tria, additional_refinements_around_source, 1.5 * r_0);
       };
 
+    // triangulation
     GridUtilities::create_triangulation_with_multigrid<dim>(grid,
                                                             this->mpi_comm,
                                                             this->param.grid,
                                                             this->param.involves_h_multigrid(),
                                                             lambda_create_triangulation,
                                                             {});
 
+    // triangulation for hanging nodes
+    if(additional_refinements_around_source > 0)
+    {
+      GridUtilities::create_coarse_triangulations_after_coarsening_and_refinement(
```

Comment:
Looking into the implementation, I believe this function does essentially the same (for `TriangulationType::Distributed`) as `GridUtilities::create_coarse_triangulations()`, a function that should actually already be called automatically by the above function called in line 540, `GridUtilities::create_triangulation_with_multigrid<dim>()`, as long as the two parameters `this->param.grid.create_coarse_triangulations` and `this->param.involves_h_multigrid()` contain valid data. Hence, I believe we need to better understand the behavior.

@jh66637 Could you check if the function `GridUtilities::create_triangulation_with_multigrid<dim>()` called in line 540 enters the two if-statements `if(involves_h_multigrid)` and (inside this if-statement) `if(data.create_coarse_triangulations)`?